src/site/includes/metatags.liquid

```diff
@@ -31,8 +31,13 @@
 <meta content="{{ tag }}" property="article:tag">
   {% endfor %}
 {% endif %}
+{% if keywords %}
+<meta name ="keywords" content="{{keywords}}">
+{% endif %}
 {% if title === 'Home' %}
 <title>VA.gov</title>
+{% elseif title === 'Find VA Locations | Veterans Affairs' %}
+<title>{{ title }}</title>
```

Comment:
I think the `| Veteran Affairs` piece should be site-wide. Have you read through this comment, https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17409#issuecomment-480360100? I could be misreading it.